plugins/scaffolder/api-report.md

```diff
@@ -110,6 +117,27 @@ export type FieldExtensionOptions<
   validation?: CustomFieldValidator<TFieldReturnValue>;
 };
 
+// @public
+export type GetProps<T> = T extends LayoutOptions
```

Comment:
What do these types give us? Not sure of the value of them? As the props wont be forwarded right?